dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -10,6 +10,9 @@ Get a tracked entity with given UID.
 
 ### `getTrackedEntityByUid.parameter.program`
```

Comment:
You will need to duplicate this text in `### *.parameter.TrackedEntityRequestParams.program` as its also a param for the collections endpoint.